application/config/constants.php

```diff
@@ -236,3 +236,9 @@
 
 // We dont need these variables any more.
 unset($base_uri, $base_url);
+
+// Turn cookiePolicy script ON or OFF. True = ON, False = OFF. And define script url.
+define('COOKIE_POLICY_ENABLED', getenv('COOKIE_POLICY_ENABLED')?getenv('COOKIE_POLICY_ENABLED'): true);
```

Comment:
Quando o PHP ler alguma variável de ambiente o seu valor sempre será uma `string`. Quando o PHP fizer a coerção de tipo na linha `117` ele sempre vai considerar como `TRUE` todo e qualquer valor que não for `vazio` e não for `0`, `[]` ([ex](https://www.php.net/manual/pt_BR/function.boolval.php)).

Em resumo se nós setarmos a variável como `FALSE` o banner será habilitado [1].

No exemplo [2] nós utilizamos uma função do [PHP que aplica um filtro](https://www.php.net/manual/pt_BR/function.filter-var.php) e transforma o valor em um booleano de fato ([o PHP tem várias constantes de filtro](https://www.php.net/manual/pt_BR/filter.constants.php)).

# Exemplos
```php
// [1] - Falso positivo
if ("FALSE") {
    echo "Sou !TRUE;";
}

// Sou !TRUE;
```

```php
// [2] - Aplicando filtro de valores com a função filter_var
define("COOKIE_POLICY_ENABLED", getenv("COOKIE_POLICY_ENABLED") ? filter_var(getenv("COOKIE_POLICY_ENABLED"), FILTER_VALIDATE_BOOLEAN) : TRUE);

if (COOKIE_POLICY_ENABLED) {
    echo "Banner habilitado";
}
```